src/views/Gallery.vue

```diff
@@ -0,0 +1,145 @@
+<template>
+    <div id="gallery-content">
+        <div id="header-mobile">
+            <div>
+                <h2>Specimen gallery</h2>
+                <h3>Self similarity telescope</h3>
+            </div>
+            <div type="button" id="change-button">
+                <span
+                    alt="Swap icon"
+                    id="change-icon"
+                    class="material-icons-sharp"
+                    >swap_horiz</span
+                >
+                <p id="change-text">Change Visualizer</p>
+            </div>
+        </div>
+
+        <h1 id="header">Specimen gallery</h1>
+
+        <div type="button" class="visualizer-bar">
+            <h2>Self similarity telescope</h2>
+            <span class="material-icons-sharp">keyboard_arrow_up</span>
+        </div>
+        <div class="gallery">
+            <SpecimenCard :seqName="sequenceName" :cardName="cardName" />
+            <SpecimenCard :seqName="sequenceName" :cardName="cardName" />
+            <SpecimenCard :seqName="sequenceName" :cardName="cardName" />
+            <SpecimenCard :seqName="sequenceName" :cardName="cardName" />
+            <SpecimenCard :seqName="sequenceName" :cardName="cardName" />
+            <SpecimenCard :seqName="sequenceName" :cardName="cardName" />
+            <SpecimenCard :seqName="sequenceName" :cardName="cardName" />
+            <SpecimenCard :seqName="sequenceName" :cardName="cardName" />
+        </div>
+
+        <div type="button" class="visualizer-bar">
+            <h2>Self similarity telescope</h2>
+            <span class="material-icons-sharp">keyboard_arrow_up</span>
+        </div>
+        <div class="gallery">
+            <SpecimenCard :seqName="sequenceName" :cardName="cardName" />
+            <SpecimenCard :seqName="sequenceName" :cardName="cardName" />
+            <SpecimenCard :seqName="sequenceName" :cardName="cardName" />
+            <SpecimenCard :seqName="sequenceName" :cardName="cardName" />
+            <SpecimenCard :seqName="sequenceName" :cardName="cardName" />
+            <SpecimenCard :seqName="sequenceName" :cardName="cardName" />
+            <SpecimenCard :seqName="sequenceName" :cardName="cardName" />
+            <SpecimenCard :seqName="sequenceName" :cardName="cardName" />
+        </div>
+    </div>
+</template>
+
+<script setup lang="ts">
+    import SpecimenCard from '../components/SpecimenCard.vue'
+
+    const sequenceName = '2-acid val of Z'
+    const cardName = 'Pinstripe'
+</script>
+
+<style lang="scss" scoped>
+    p,
+    h1,
+    h2,
+    h3 {
+        margin: 0;
+    }
+    h1 {
+        font-size: var(--ns-size-title);
+    }
+    h2 {
+        font-size: var(--ns-size-heading);
+    }
+    h3 {
+        font-size: var(--ns-size-subheading);
+    }
+
```

Comment:
Should these declarations of the sizes of the various heading levels be global? They look pretty much like they should be the defaults throughout frontscope, especially given the names of the CSS vars. Or is there some reason they need to be specific to the Gallery? If not, please move them into a global place and remove them here.

Reply:
This has been made local to prevent accidental re-styling of elements throughout the other parts of the website. Ideally, we do want to make it global eventually but at the moment this conflicts with other development so we would like to keep it local for now.